lldb/unittests/tools/lldb-server/tests/TestClient.h

```diff
@@ -20,6 +20,10 @@
 #include <optional>
 #include <string>
 
+#ifdef SendMessage
+#undef SendMessage
+#endif
+
```

Comment:
I guess that means we're including a windows header somewhere. Any chance to avoid that? Llvm is pretty strict about that. Lldb is not, though I think it's a good practice.

Reply:
Note TestClient.cpp and LLGSTest.cpp already contain this code. Windows headers are necessary for socket definitions. It is impossible to separate them. This fix is necessary because the order of headers has been changed. Note we can remove the same code in TestClient.cpp and LLGSTest.cpp. I just minimized changes. Should I remove it from TestClient.cpp and LLGSTest.cpp?

Reply:
I will retest it after #104439. Probably we won't need it.

Reply:
If it's just due to header moves that's fine.

It's definitely possible to isolate the windows headers. Lllvm does that by making sure it only includes the system headers in implementation files (of the library wrapping them). It'd be nice to have lldb do the same thing, but we're pretty far off.